### PR TITLE
Story

### PR DESCRIPTION
**Pull request by Tudiman**
Story + dialogue box update, storyline json is about 30% completed.

### CODE REVIEW

File: css/dbox.css [hunk 1]
```diff
@@ -20,8 +19,8 @@
     border-style: outset;
 
     display: grid;
-    
-    font-family: "8bitDragon", Arial, sans-serif;
+
+    font-family: Cabin;
```
Comment (andreivladutz) [inline css/dbox.css]: I really liked that font :(

File: js/StoryClasses/StoryParser.js [hunk 2]
```diff
@@ -17,60 +17,85 @@ class Dialogue {
...
 
-            let quest = StoryParser.selfReference.quests[npc_id];
-            if(quest.stage === 0 && quest.stage === 1) {
+            let quest = this.quests[npc_id];
+            console.log(quest);
+            if(quest.stage === 0 || quest.stage === 1) {
                 if(type === 1) {
                     quest.prevStage = 3;
                     quest.stage = 3;
+                    localStorage.setItem(`questStages|${npc_id}`,{stage: quest.stage, prevStage: quest.prevStage});
```
Comment (andreivladutz) [inline js/StoryClasses/StoryParser.js]: Added StateSaver class which saves and loads game state from the localStorage in recent commits on the branch I'm currently working on. Would be nice if we moved this logic there sometime in the future.

File: css/dbox.css (hunk 1 above)
Comment (Tudiman) [reply css/dbox.css]: Go swivel.

File: js/StoryClasses/StoryParser.js (hunk 2 above)
Comment (Tudiman) [reply js/StoryClasses/StoryParser.js]: Soon™

**Comment by Tudiman** [comment]
Wowow, it took less than 48 hours to get greenlight on merge, nice. Reviews really do work.

**Pull request merged**